ui/src/components/sidebar.tsx

```diff
@@ -32,6 +32,7 @@ interface SidebarNavProps {
       params?: Record<string, string | undefined>;
       title: string;
       visible?: boolean; // default: true
+      disabled?: boolean; // default: false
```

Comment:
Would this be a little simpler if the `to` param was optional, and the link is disabled if it is not given? It may confuse someone that they have to give an URL even if the link is disabled.

Reply:
The change to Sidebar params should also probably be a separate commit.

Reply:
Oh, of course, was going to do that but I forgot.

Reply:
@mmurto let me still fix this so the "disabled" doesn't need to be given.